cmd/ps.go

```diff
@@ -56,7 +56,9 @@ func PsScale(appID string, targets []string) error {
 			fmt.Printf("'%s' does not match the pattern 'type=num', ex: web=2\n", target)
```

Comment:
I think we should just make this return an error if there's a bad seed in the scale list.